### ISSUE TITLE
Cannot Log Into Any App On browser v0.21.6

### ISSUE THREAD

**Issue by jehunter5811** [blockstack/blockstack-browser#1142]
I can't tell is this something local to me or impacting everyone, but after downloading the pre-release browser v0.21.6, I cannot log into any Blockstack apps. I am using Chrome and I get the following console error: 

```
Uncaught (in promise) TypeError: Cannot read property 'split' of undefined
    at isLaterVersion (main.js:50534)
    at AuthModal.componentWillReceiveProps (main.js:3889)
    at ReactCompositeComponentWrapper.updateComponent (main.js:153895)
    at ReactCompositeComponentWrapper.receiveComponent (main.js:153828)
    at Object.receiveComponent (main.js:159835)
    at ReactCompositeComponentWrapper._updateRenderedComponent (main.js:154035)
    at ReactCompositeComponentWrapper._performComponentUpdate (main.js:154005)
    at ReactCompositeComponentWrapper.updateComponent (main.js:153926)
    at ReactCompositeComponentWrapper.performUpdateIfNecessary (main.js:153842)
    at Object.performUpdateIfNecessary (main.js:159867)
```
Here are two screenshots of the full log: 

<img width="1363" alt="screen shot 2017-12-28 at 10 19 37 am" src="https://user-images.githubusercontent.com/10519834/34416354-b4009ed8-ebb8-11e7-9205-1e5c09f4f78e.png">
<img width="1366" alt="screen shot 2017-12-28 at 10 19 50 am" src="https://user-images.githubusercontent.com/10519834/34416355-b5fea784-ebb8-11e7-94f5-e0863d42603e.png">

**Comment by pranavj1001** [comment]
even I'm facing the same problem. I fixed it by downgrading to v0.21.5

**Comment by dantrevino** [comment]
Same issue.  I cannot log into apps with 0.21.6.  Using Linux.

GET http://localhost:6270/v1/names/ net::ERR_CONNECTION_REFUSED
(anonymous)
proxyFetchForSomeHosts @ main.js:18327
login @ main.js:3941
invokeGuardedCallback @ main.js:156628
executeDispatch @ main.js:150730
executeDispatchesInOrder @ main.js:150750
executeDispatchesAndRelease @ main.js:150165
executeDispatchesAndReleaseTopLevel @ main.js:150176
forEachAccumulated @ main.js:161919
processEventQueue @ main.js:150376
runEventQueueInBatch @ main.js:156695
handleTopLevel @ main.js:156705
handleTopLevelImpl @ main.js:156780
perform @ main.js:161243
batchedUpdates @ main.js:156468
batchedUpdates @ main.js:159409
dispatchEvent @ main.js:156855
auth:1 Uncaught (in promise) TypeError: Failed to fetch
Promise rejected (async)
login @ main.js:3945
invokeGuardedCallback @ main.js:156628
executeDispatch @ main.js:150730
executeDispatchesInOrder @ main.js:150750
executeDispatchesAndRelease @ main.js:150165
executeDispatchesAndReleaseTopLevel @ main.js:150176
forEachAccumulated @ main.js:161919
processEventQueue @ main.js:150376
runEventQueueInBatch @ main.js:156695
handleTopLevel @ main.js:156705
handleTopLevelImpl @ main.js:156780
perform @ main.js:161243
batchedUpdates @ main.js:156468
batchedUpdates @ main.js:159409
dispatchEvent @ main.js:156855

**Comment by dantrevino** [comment]
I just tried a brand-new, clean install and was unable to replicate.

**Comment by yknl** [comment]
A possible cause of this error is that the app you are logging into is using an older version of `blockstack.js`. The yeoman Blockstack app generators currently generate apps that have this problem.

**Comment by jehunter5811** [comment]
@dantrevino I just tried a clean install of 0.21.6 and I'm still hitting the same issue. 

@yknl I'm testing auth with this version on the todo app, the hello, blockstack app, and my own app (which is running blockstack.js 0.14.2).

**Comment by jehunter5811** [comment]
Ah, quick follow up on this. Initial log in fails, but if I refresh and approve the sign in again, it works.

**Comment by larrysalibra** [comment]
Thanks for letting us know about this @jehunter5811 

A fix for this will ship in v0.22.0!
